ui/.env.development

```diff
@@ -5,3 +5,4 @@ REACT_APP_SENTRY_DSN=https://10930d3b0a8d427cad2147d6d845be56@o226447.ingest.sen
 REACT_APP_SENTRY_ENABLED=false
 # REACT_APP_RELEASE_VERSION is set in config-overrides.js
 REACT_APP_RELEASE_VERSION=unknown
+REACT_APP_MIXPANEL_TOKEN=
```

Comment:
@baurine Could you provide the token for this?

Reply:
We can get it in the history commit.